index.js

```diff
@@ -1,6 +1,8 @@
+const eventDenyList = ['$autocapture', 'organization usage report']
```

Comment:
A comment would be great so that we know what this is about in the future if we are doing migrations etc.